src/test_c2s_srv.c

```diff
@@ -548,12 +619,18 @@ int test_c2s(Connection *ctl, tcp_stat_agent *agent, TestOptions *testOptions,
     // Set up the FD_SET and activeStreams for the next select.
     activeStreams = connections_to_fd_set(c2s_conns, streamsNum, &rfd, &max_fd);
   }
-
   measured_test_duration = secs() - start_time;
+  // From the NDT spec:
   //  throughput in kilo bits per sec =
   //  (transmitted_byte_count * 8) / (time_duration)*(1000)
   *c2sspd = (8.0e-3 * bytes_read) / measured_test_duration;
 
+  // As a kindness to old clients, drain their queues for a second or two.
+  // TODO: Fix client code to eliminate the need for this.  Clients that need
+  //       this line should be removed from their respective teme pools and
```

Comment:
Suggest rephrasing. I've never herad of that and I don't know that I'd be able to find that:

![image](https://cloud.githubusercontent.com/assets/7783288/12055071/59ae7950-aef7-11e5-9add-b5e2fbf220b3.png)